omnibus/config/projects/chef-server.rb

```diff
@@ -46,6 +46,7 @@
 dependency "openresty-lpeg"  # lua-based routing
 dependency "runit"
 dependency "chef_backup-gem" # chef-server-ctl backup
+dependency "rest-client-gem" # chef-server-ctl server-admins commands
```

Comment:
This is a larger problem than just this PR, but I want to get the idea out there. What needs the `rest-client` gem, and why doesn't it fetch it for itself? It feels like an anti-pattern to me the more that we specify the gem dependencies of tools here in top level of the project. At the very minimum, that should be `chef-server-ctl.rb` that defines the deps, and eventually have a Gemfile / Gemfile.lock there to handle this.